canonical/ubuntu.com
# WD-17753 Iot embedded modal form - removed pagination + conversion to single page

## Done

- Removed pagination
- Conversion to single page
- Cleanup

## QA

- Check the old modal here first: https://ubuntu.com/embedded#get-in-touch
- Check the new modal at: https://ubuntu-com-14567.demos.haus/embedded#get-in-touch
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check that modal form is a single page without pagination.
- Submit the form to make sure it's working.

## Issue / Card

Fixes #[WD-17753](https://warthogs.atlassian.net/browse/WD-17753)


[WD-17753]: https://warthogs.atlassian.net/browse/WD-17753?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
